service/src/main/kotlin/io/provenance/onboarding/domain/usecase/cee/common/client/CreateClient.kt

```diff
@@ -0,0 +1,56 @@
+package io.provenance.onboarding.domain.usecase.cee.common.client
+
+import io.provenance.core.KeyType
+import io.provenance.onboarding.domain.usecase.AbstractUseCase
+import io.provenance.onboarding.domain.usecase.cee.common.client.model.CreateClientRequest
+import io.provenance.onboarding.domain.usecase.common.originator.GetOriginator
+import io.provenance.scope.encryption.model.DirectKeyRef
+import io.provenance.scope.encryption.util.toJavaPrivateKey
+import io.provenance.scope.encryption.util.toJavaPublicKey
+import io.provenance.scope.sdk.Affiliate
+import io.provenance.scope.sdk.Client
+import io.provenance.scope.sdk.ClientConfig
+import io.provenance.scope.sdk.SharedClient
+import java.net.URI
+import java.security.KeyPair
+import java.util.concurrent.TimeUnit
+import org.springframework.stereotype.Component
+
+@Component
+class CreateClient(
+    private val getOriginator: GetOriginator
+) : AbstractUseCase<CreateClientRequest, Client>() {
+    override suspend fun execute(args: CreateClientRequest): Client {
+        val originator = getOriginator.execute(args.account.originatorUuid)
+        val affiliate = Affiliate(
+            signingKeyRef = DirectKeyRef(KeyPair(originator.keys[KeyType.SIGNING_PUBLIC_KEY].toString().toJavaPublicKey(), originator.keys[KeyType.SIGNING_PRIVATE_KEY].toString().toJavaPrivateKey())),
+            encryptionKeyRef = DirectKeyRef(KeyPair(originator.keys[KeyType.ENCRYPTION_PUBLIC_KEY].toString().toJavaPublicKey(), originator.keys[KeyType.ENCRYPTION_PRIVATE_KEY].toString().toJavaPrivateKey())),
```

Comment:
Nit: Is it time we extract these super-long calls throughout the project to a utility function, as @mnguyen-figure previously suggested?

Reply:
done :D updated key-access-lib to provide a mechanism to store these in a map + added first class citizens for retrieving the common key types.